docs/Protected_build_porting_guide.md

```diff
@@ -0,0 +1,363 @@
+# Protected Build Porting Guide
+
+
+## Contents
+- [Introduction](#introduction)
+	- [Prerequisites](#prerequisites)
+	- [What is Protected Build](#what-is-protected-build)
+- [Steps for porting protected build in a new chipset](#steps-for-porting-protected-build-in-a-new-chipset)
+	- [Create a new configuration for Protected Build](#create-a-new-configuration-for-protected-build)
+	- [Creating linker scripts](#creating-linker-scripts)
+	- [Userspace Initialization](#userspace-initialization)
+	- [MPU Initialization](#mpu-initialization)
+	- [Allocation of kernel and user heaps](#allocation-of-kernel-and-user-heaps)
+	- [Changes in Make files](#changes-in-make-files)
+	- [Changes in the download method](#changes-in-the-download-method)
+
+
+## Introduction
+This is a guide for porting Protected Build for a new chipset.
```

Comment:
Please remove the word porting from all places in the document. 
Instead you can say "This is a guide for supporting TizenRT protected build on a new chipset"

Reply:
Done